### ISSUE TITLE
As a WebWorks developer, I can use the hasCapability function to determine what the device is capable of

### ISSUE THREAD

**Issue by nukulb** [blackberry-webworks/BB10-Webworks-API#24]
This user story is a placeholder to implement true querying of the device capabilities, so that the return values are relevant for different platforms instead of hardcoded as for Beta 1

**Comment by nukulb** [comment]
@kwallis - this needs more details if there is anything to do here, let me know your thoughts

**Comment by kwallis** [comment]
Yes, we do need to do this, but it isn't top priority.  What details would you need?  We don't need to change or adjust the current list of capabilities, we just need to change the implementation to query the system rather than being hardcoded.
